### DISCUSSION TITLE
How to position messages in the bottom-right corner?

### DISCUSSION THREAD

**Discussion by Nullarity** [folke/noice.nvim#676] [Q&A]
Hi,
Could you please guide me on how to change the position of the message output? Currently, it appears in the top-right. How can I adjust it to the bottom-right?
Thank you!

**Comment by chojs23** [comment]
You should set nvim-notify options. 
```
     require("notify").setup({
        top_down = false
      })
```

**Reply by PlexSheep** [reply]
This works! Thank you.
![Screenshot_20240422_173115](https://github.com/folke/noice.nvim/assets/58274330/7d0f401f-a308-4f20-8490-c4730785a2f9)

**Comment by Nullarity** [comment]
Thank you!
I see that new messages are added on top of the old ones. Is there a way to keep new messages at the bottom and move the old ones to the top?
